doc/hoomd-examples.rst

```diff
@@ -141,3 +141,31 @@ format. In this example, type 'A' has 3 vertices (the first 3 in the list) and t
         print(s.state['hpmc/convex_polygon/vertices'])
 
 Access read state data in the same way.
+
+Access logged data
+^^^^^^^^^^^^^^^^^^
+
+.. ipython:: python
+
+    with gsd.hoomd.open(name='example.gsd', mode='wb') as t:
+        s = gsd.hoomd.Snapshot()
+        s.particles.N = 4
+        s.log['particles/net_force'] = [[-1,2,-3],
```

Comment:
I think this data is being saved as an array of integers, based on the formatting -- this choice of an example array might be a little confusing to a user who is expecting to read/write float values for forces. I would change the values to include decimals or create a NumPy array with the appropriate type.